core/channel.go

```diff
@@ -193,6 +196,9 @@ func logChannelStates(src, dst *ProvableChain, srcChan, dstChan *chantypes.Query
 
 func checkChannelFinality(src, dst *ProvableChain, srcChannel, dstChannel *chantypes.Channel) (bool, error) {
 	logger := GetChannelPairLogger(src, dst)
+	if src.Chain.Path().ChannelID == "" && dst.Chain.Path().ChannelID == "" {
+		return true, nil
+	}
```

Comment:
I'd like to comment same as for checkConnectionFinality.

Reply:
Removed